content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -262,6 +262,20 @@ This annotation is part of the Kubernetes Resource Model (KRM) Functions Specifi
 which is used by Kustomize and similar third-party tools.
 For example, Kustomize removes objects with this annotation from its final build output.
 
+### config.kubernetes.io/apply-time-mutation
+
+Type: Annotation
+
+Example: `config.kubernetes.io/apply-time-mutation: "true"`
```

Comment:
Are you sure it's value should be in boolean? 
Please read : https://github.com/kubernetes-sigs/cli-utils/blob/master/README.md#apply-time-mutation
Let's correct the values first

Reply:
Hi @Ritikaa96, I am getting confused with the example value for config.kubernetes.io/apply-time-mutation annotation is a multiline string in YAML format, specifying a list of substitutions, each represented by a dictionary (or map). Each substitution specifies the sourceRef, sourcePath, targetPath, and optionally the token. So how do I mention that out or do I simply leave blank example value.
```
metadata:
  annotations:
    config.kubernetes.io/apply-time-mutation: |
      - sourceRef:
          kind: Pod
          name: pod-b
        sourcePath: $.status.podIP
        targetPath: $.spec.containers[?(@.name=="nginx")].env[?(@.name=="SERVICE_HOST")].value
        token: ${pod-b-ip}
      - sourceRef:
          kind: Pod
          name: pod-b
        sourcePath: $.spec.containers[?(@.name=="nginx")].ports[?(@.name=="tcp")].containerPort
        targetPath: $.spec.containers[?(@.name=="nginx")].env[?(@.name=="SERVICE_PORT")].value
        token: ${pod-b-port}
```
In this example, the multiline string contains two substitutions, each represented by a dictionary (indicated by the - before each item). Each dictionary specifies the sourceRef, sourcePath, targetPath, and token properties, which are necessary for the Applier to perform the field value substitution during the apply process. The ${pod-b-ip} and ${pod-b-port} are placeholders that will be replaced with actual values based on the source object pod-b during the apply process.